CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 ## 3.87.0 (Unreleased)
+### Added
```

Comment:
new line?

Reply:
Created this PR using the tool, it added it automatically here.